docs/UserGuide.md

```diff
@@ -109,16 +109,22 @@ Clears all entries from the address book.
 
 Format: `clear`
 
+> [!WARNING]
+> After confirmation, any cleared data or contact cannot be retrieved. This action is IRREVERSIBLE.
+
+* Recommended practice: Always have a copy of current data (refer to the export feature). You can use import feature if you have lost your data after clearing it unintentionally.
```

Comment:
Can include hyperlink to `Exporting data` section and `Importing data` in a similar format to how `import` feature section has hyperlinks